introduction.md

```diff
@@ -1,3 +1,7 @@
 ﻿Hi, let me introduce myself; I'm Vicke.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?

Reply:
Ok, if you say so!